docs/csharp/language-reference/operators/null-conditional-operators.md

```diff
@@ -33,22 +33,24 @@ Used to test for null before performing a member access (`?.`) or index (`?[`) o
 int? length = customers?.Length; // null if customers is null   
 Customer first = customers?[0];  // null if customers is null  
 int? count = customers?[0]?.Orders?.Count();  // null if customers, the first customer, or Orders is null  
-  
 ```  
   
 ```vb  
-Dim length = customers?.Length  ‘’ null if customers is null  
-Dim first as Customer = customers?(0);  ‘’ null if customers is null  
-Dim count as Integer? = customers?[0]?.Orders?.Count();  // null if customers, the first customer, or Orders is null  
-  
+Dim length = customers?.Length  ' null if customers is null  
+Dim first as Customer = customers?(0)  ' null if customers is null  
+Dim count as Integer? = customers?(0)?.Orders?.Count()  ' null if customers, the first customer, or Orders is null  
 ```  
   
  The last example demonstrates that the null-condition operators are short-circuiting.  If one operation in a chain of conditional member access and index operation returns null, then the rest of the chain’s execution stops.  Other operations with lower precedence in the expression continue.  For example, `E` in the following always executes, and the `??` and `==` operations execute.  
   
-```vb-c#  
+```csharp
```

Comment:
Those are not valid C# statements, so semicolons don't make sense here. I think those are just supposed to be two separate expressions.

Do you think it would be clearer if they were expanded to statements, something like:

```c#
var maybeE = A?.B?.C?[0] ?? E;
var equalsE = A?.B?.C?[0] == E;